src/lib/Components/Bidding/Screens/ConfirmBid/index.tsx

```diff
@@ -425,7 +428,7 @@ export class ConfirmBid extends React.Component<ConfirmBidProps, ConfirmBidState
               <Timer liveStartsAt={sale.live_start_at} endsAt={sale.end_at} />
             </Flex>
 
-            <View>
+            <Box>
```

Comment:
Out of curiosity, why the change to `Box`?

Reply:
We generally avoid `<div>` or `<View>` that are platform-specific and use `<Box>` (which is universal / copy / pasteable to RN).